tsconfig.cjs.json

```diff
@@ -9,6 +9,7 @@
     "esModuleInterop": true,
```

Comment:
If you use rollup. I think you can make `tsconfig` easy.

Just use single file and use `tsconfigOverride` to change the config.

Reply:
All right. I'm gonna change the build process to a more elegant

Reply:
I committed tscconfig integration Please check.